level00/CoffeeMachineTests.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE RankNTypes #-}
+module CoffeeMachineTests (propertyTests) where
+
+import Control.Lens (Prism', prism, matching, preview, review)
+
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.Hedgehog (testProperty)
+
+import Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+-- Starting
+addTen :: Int -> Int
+addTen = (+10)
+
+prop_addTen :: Property
+prop_addTen = property $ do
+  n <- forAll $ Gen.int (Range.linear 0 1000)
+  addTen n === n + 10
+
+badReverse :: [a] -> [a]
```

Comment:
Should have clear instructions in the comments, like "This function is broken. When you run the tests, it should show you a breaking case. After you have run the tests, fix this function."